src/Mapping/PropertyMapping.php

```diff
@@ -82,13 +83,13 @@ public function setFile(object $obj, File $file): void
     /**
      * Gets the fileName property of the given object.
      *
-     * @param object $obj The object
+     * @param object $obj The object or array
```

Comment:
The phpdoc type should be "object|array" (here and in all other points where is relevant)